src/applications/financial-status-report/pages/resolution/resolutionOption.js

```diff
@@ -10,7 +9,6 @@ export const uiSchema = {
       'ui:validations': [validateResolutionOption],
       resolutionOption: {
```

Comment:
I'm a nervous bee, but do we want to use `ui:options` and explicitly set `hideOnReview: true,` ?
Deffo not merge blocking, just a thought!